### PR TITLE
LPS-183757 Center publication modal entry titles

### PR DESCRIPTION
**Pull request by ces-quanhuynh**
Hi @ces-luannguyen 
This PR is for fixing https://issues.liferay.com/browse/LPS-183757
Please review and leave your comment.
Thanks,
Quan

### CODE REVIEW

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js [hunk 1]
```diff
@@ -931,12 +931,16 @@ export default function ChangeTrackingIndicator({
...
+				{!!entry.description && (
+					<ClayList.ItemText subtext>
+						{entry.description}
+					</ClayList.ItemText>
+				)}
 			</ClayList.ItemField>
 		);
 
+		console.log(!itemField);
+
```
Comment (ces-luannguyen) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js]: Please remove this.

Comment (ces-quanhuynh) [reply modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js]: My bad! I've removed it.

**Comment by ces-quanhuynh** [comment]
@ces-luannguyen 
I've updated the PR. 
Please review again!
Thanks!

**Review by ces-luannguyen — APPROVED** [review]

**Comment by ces-luannguyen** [comment]
Forwarded https://github.com/liferay-echo/liferay-portal/pull/12043

**Pull request closed without merging**